src/rt-app_parse_config.c

```diff
@@ -890,19 +923,41 @@ parse_tasks(struct json_object *tasks, rtapp_options_t *opts)
 
 	log_info(PFX "Parsing tasks section");
 	opts->nthreads = 0;
+	opts->nzthreads = 0;
 	foreach(tasks, entry, key, val, idx) {
 		instance = get_int_value_from(val, "instance", TRUE, 1);
 		opts->nthreads += instance;
+		if (!instance)
+			opts->nzthreads++;
 	}
 
 	log_info(PFX "Found %d tasks", opts->nthreads);
-	opts->threads_data = malloc(sizeof(thread_data_t) * opts->nthreads);
+	log_info(PFX "Found %d ghost-tasks", opts->nzthreads);
```

Comment:
nzthreads as in zero? :-)

Reply:
I definitely could have come up with a better name :)

Any suggestions? Maybe nzinstances or nzeroinstances?

Reply:
Name is actually fine with me.
Maybe stick somewhere a comment so that we don't forget what this variable is about?
Guess should be consinsted with how you call them in log_info though (nitpick FTW!).

Reply:
Does this look better?

`log_info(PFX "Found %d \'\"instance\": 0\' tasks", opts->nzthreads);`

It should be self documented and no need for extra comments I believe.